maestro/common/subflow-skip-chrome-welcome.yaml

```diff
@@ -1,6 +1,13 @@
 appId: ${APP_ID}
 ---
 ####### Bypass Chrome on-boarding screen #######
+# If no default browser selected, select Chrome
+- tapOn:
+    text: ".*Chrome.*"
+    optional: true
+- tapOn:
+    text: ".*ALWAYS.*"
+    optional: true
```

Comment:
Bitrise emulators have no default emulator set. This ensures Chrome is selected when the app picker shows up.